src/helpers/helpers.chart.js

```diff
@@ -21,6 +21,25 @@ export function scaleValue(scale, value, fallback) {
   return isFinite(value) ? scale.getPixelForValue(value) : fallback;
 }
 
+/**
+ * @param {Object} scales - chartjs object with all scales
+ * @param {Object} options - plugin options
+ * @param {string} key - annotation plugin scale id option key
+ * @returns {string} the unique scale defined in chartjs or the key passed as argument
+ */
+export function retrieveScaleID(scales, options, key) {
+  const scaleID = options[key];
+  if (scaleID || key === 'scaleID') {
+    return scaleID;
+  }
+  const axis = key.charAt(0);
+  const axes = Object.values(scales).filter((scale) => scale.axis && scale.axis === axis);
+  if (axes.length === 1) {
```

Comment:
Why not just default to the first scale when there are multiple and user did not specify which one to use?

Reply:
The check can be removed and returns always  `return axes[0].id;`. Anyway we must check if there is not any scale,